File: Algorithm/Other/at coder/Other/A36/A36.js
function main(input) {
	const [N, K] = input.trim().split(" ").map(Number);
	const minSteps = 2 * (N - 1);

	if (K >= minSteps && (K - minSteps) % 2 === 0) {
		console.log("Yes");
	} else {
		console.log("No");
	}
}

// 以下はNode.jsでの標準入力処理
require("readline")
	.createInterface({
		input: process.stdin,
	})
	.on("line", (line) => {
		main(line);
	});

File: Algorithm/Other/at coder/Other/A36/A36.py
def main():
    N, K = map(int, input().split())
    min_steps = 2 * (N - 1)

    if K >= min_steps and (K - min_steps) % 2 == 0:
        print("Yes")
    else:
        print("No")

if __name__ == "__main__":
    main()
